### PR TITLE
59: Update header design and layout

### LINKED ISSUE #59: Update headings in search and taxonomy listings based on new design
**As a user of the Travel theme, when I search or visit a page, have context to what kind of page I'm on, what I just did and how to get back to where I just was.**

This is a placeholder ticket to implement design. Currently, Maxim to update Figma mocks with input from @jwold: https://www.figma.com/file/L14jTE7iNGOE7DH57Ro7ncyu/Travel-Theme

Whatever changes will happen will be added here.

Currently blocked by this design ticket: https://xwp-co.atlassian.net/browse/AP2-40
I'll update the status of this issue with the appropriate ACs when AP2-40 is complete.

### PR DESCRIPTION
**Pull request by miina**
Fixes #59.
- Uses header with search for archives and adventures post type.
- Adds breadcrumbs to taxonomy page;
- Adds title to search page.

Todo:
- Styling fixes

The search page looks like this:
screen shot 2018-04-24 at 2 48 18 pm

However, the taxonomy and adventure page look like this and need styling fixes -- for some reason any change made to the adventure.css or archive.css breaks the style, the style fixes are added only to search.css
screen shot 2018-04-24 at 2 48 03 pm

### CODE REVIEW

**Comment by miina** [comment]
@kienstra This PR should be ready other than styling issues.

@mehigh Do you think you could review the header, breadcrumbs, and the title's style? The search page needs just a review but adventure and archives page need styling fixes, too -- for some reason here as well as in #64, any change I make to `adventure.css` or `archive.css` breaks the style, would be great if you could help with that.

**Comment by miina** [comment]
Edit: Looks like `archive.css` looks better after the last change, however, the `adventure.css` is completely stripped if I make any changes, so this still needs some attention. Currently the header looks like this:
screen shot 2018-04-24 at 7 29 26 pm

Basically the difference with other pages is missing this:
```
.search-travel-btn,
.search-travel-btn:visited{
    margin-left: 25px;
    color: #8b58e3;
}
.travel-date-input{
    line-height:0.125rem
}
```

Not sure why this breaks the CSS on `adventure.css`.

(cc @mehigh)

File: search.php [hunk 1]
```diff
@@ -14,6 +14,9 @@
 		<section class="travel-results pb1 md-pt1">
 			<div class="travel-inline-list travel-results-list">
 				<div class="max-width-3 mx-auto px1 md-px2">
+					<?php if ( isset( $_GET['s'] ) ) : ?>
+						<h1><?php printf( '%d Results for <strong>%s</strong>', esc_html( amp_travel_get_posts_count() ), esc_html( $_GET['s'] ) ); ?></h1>
```
Comment (kienstra) [inline search.php]: The search template is looking good. How about using `_n()` to differentiate between singular and plural here?

File: templates/breadcrumb.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+if ( ! is_tax() ) {
+	return;
+}
+$term = get_term_by( 'slug', get_query_var( 'term' ), get_query_var( 'taxonomy' ) );
+?>
+<div id="crumb" class="py2 pl2 pr2 border-bottom travel-border-gray">
+	<span class="bold py1 pl1 pr1"><a class="travel-link" href="<?php echo esc_html( home_url() ); ?>"><?php esc_html_e( 'Home' ); ?></a></span>
+	<span class="travel-results-result-subtext">></span>
+	<span class="bold py1 pl1 pr1"><?php echo esc_html( $term->name ); ?></span>
```
Comment (kienstra) [inline templates/breadcrumb.php]: The breadcrumbs look really good. But I think the second item could be changed:

breadcrumb-taxonomy

File: templates/breadcrumb.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * @package WPAMPTheme
+ */
+
+if ( ! is_tax() ) {
+	return;
+}
+$term = get_term_by( 'slug', get_query_var( 'term' ), get_query_var( 'taxonomy' ) );
+?>
+<div id="crumb" class="py2 pl2 pr2 border-bottom travel-border-gray">
+	<span class="bold py1 pl1 pr1"><a class="travel-link" href="<?php echo esc_html( home_url() ); ?>"><?php esc_html_e( 'Home' ); ?></a></span>
```
Comment (kienstra) [inline templates/breadcrumb.php]: Could you please change:
```php
esc_html( home_url() )
```
to:
```php
esc_url( home_url() )
```

File: archive.php [hunk 4]
```diff
@@ -10,10 +10,13 @@
 <div class="travel-overlay-fx-scale">
 	<div class="travel-no-focus flex-auto overflow-auto" role="button" tabindex="-1">
 
+		<?php get_template_part( 'templates/breadcrumb' ); ?>
 		<!-- Results -->
 		<section class="travel-results pb1 md-pt1">
 			<div class="travel-inline-list travel-results-list">
 				<div class="max-width-3 mx-auto px1 md-px2">
+					<h1><?php echo wp_kses_post( amp_travel_get_archive_title() ); ?></h1>
+					<div><?php printf( '%d adventures', esc_html( amp_travel_get_posts_count() ) ); ?></div>
```
Comment (kienstra) [inline archive.php]: How about using `_n()` in case there's only one adventure?

**Review by kienstra — CHANGES_REQUESTED** [review]
**Thanks, A Few Points**

Hi @miina,
Thanks a lot for your work here. This matches the updated [designs](https://www.figma.com/file/L14jTE7iNGOE7DH57Ro7ncyu/Travel-Theme) well, other than the styling issues that you mentioned above.

There are a few points here. But this is almost ready, pending the styling points.

**Comment by miina** [comment]
@kienstra Thanks for reviewing, the changes are made within 6f5c870.

**Comment by miina** [comment]
@kienstra and @mehigh Added the missing style change for adventure page as well now, the style just needs reviewing now (and any fixes that Mike might find necessary).

**Comment by kienstra** [comment]
**Approved, Other Than Styling Review**

Hi @miina,
Your latest changes look good. As you mentioned, this could still a look from Mike. But other than that, it's approved.

**Review by kienstra — APPROVED** [review]

**Comment by kienstra** [comment]
**Request For Retro Sanity-Check**

@mehigh, if you have time, could you do a quick sanity-check of the styling here? The styling appears well on the `archive.php`, `search.php`, and single adventure pages. 

But otherwise, I think this is ready to merge.

**Comment by mehigh** [comment]
CSS looks OK, thanks for the work here @miina !

**Pull request merged**